dev/release/utils-binary.sh

```diff
@@ -41,6 +41,7 @@ docker_gpg_ssh() {
       -i "${docker_ssh_key}" \
       -p ${ssh_port} \
       -R "/home/arrow/.gnupg/S.gpg-agent:${gpg_agent_extra_socket}" \
+      -t \
```

Comment:
what does "-t" to here?

Reply:
This is for suppressing warnings from `gpg`:

```text
warning: Could not set GPG_TTY to stdin: Inappropriate ioctl for device
```

If we add `-t` here, `ssh` allocates a tty. It suppresses the warnings.